src/app/portlets/dot-edit-page/content/components/dot-edit-content-view-as-toolbar/dot-edit-content-view-as-toolbar.component.spec.ts

```diff
@@ -222,6 +222,7 @@ describe('DotEditContentViewAsToolbarComponent', () => {
                     lockedBy: '123'
                 }
             });
+            spyOn(dotLicenseService, 'isEnterprise').and.returnValue(of(false));
```

Comment:
Why create a test just for this case?, I think is better add it into this one:

https://github.com/dotCMS/core-web/blob/8809f5c7bf995cd7e9810c529da7821ba372ebbc/src/app/portlets/dot-edit-page/content/components/dot-edit-content-view-as-toolbar/dot-edit-content-view-as-toolbar.component.spec.ts#L129

Reply:
@freddyucv  this already exist in the past, I just fixed cause it was broken. I'll see if it can be done